### PR TITLE
Feature/support password grant

### PR DESCRIPTION
**Pull request by antiline**
## Support password grant

### CODE REVIEW

File: src/lib/ridibooks/api/base.py [hunk 1]
```diff
@@ -16,25 +19,43 @@ def __init__(self, access_token: Optional[str] = None, phpsession_id: Optional[s
...
+
         else:
-            kwargs['data'] = data
+            if is_json_data:
+                kwargs['json'] = data
+            else:
+                kwargs['data'] = data
 
         try:
+            logger.info(kwargs)
```
Comment (JungleKim) [inline src/lib/ridibooks/api/base.py]: Remove this.

File: src/lib/ridibooks/api/store.py [hunk 2]
```diff
@@ -1,14 +1,57 @@
...
+            return self._request_with_internal_server_auth(
+                token_key=AuthList.ACCOUNT_TO_STORE,
+                method=HttpMethod.POST, path=self.IS_LOGINABLE, data={'u_id': username, 'password': password}, is_json_data=True
+            )
+
+        except HTTPException as e:
+            content = json.loads(e.content.decode('utf-8'))
+            if e.status == 400:
+                logger.error(f'[STORE API][IS_LOGINABLE] Error: {content}')
+                raise InvalidRequestException()
```
Comment (JungleKim) [inline src/lib/ridibooks/api/store.py]: These don't need to be a instance.

File: src/lib/ridibooks/internal_server_auth/constants.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+
+class ServiceList(BaseConstant):
+    ACCOUNT = 'account'
+    LIBRARY = 'library'
+    USER_BOOK = 'user-book'
+    BOOK = 'book'
+    CPS = 'cps'
+    SEARCH = 'search'
+    STORE = 'store'
```
Comment (JungleKim) [inline src/lib/ridibooks/internal_server_auth/constants.py]: How about remove unused value?

Comment (antiline) [reply src/lib/ridibooks/internal_server_auth/constants.py]: I considered it. This is not used now, but should be treated as a constant and must use the same value for all services.  We need to make it library.

Comment (JungleKim) [reply src/lib/ridibooks/internal_server_auth/constants.py]: Okay, I see.

**Pull request merged**